### PR TITLE
Added support for sql-formatter

### PR DESCRIPTION
**Pull request by Trisk3lion**
Support for https://github.com/sql-formatter-org/sql-formatter.

### CODE REVIEW

File: sqlformat.el [hunk 1]
```diff
@@ -71,7 +71,8 @@ This command should receive SQL input via STDIN and output the
 reformatted SQL to STDOUT, returning an appropriate exit code."
   :type '(choice (const :tag "Use \"sqlformat\"" sqlformat)
                  (const :tag "Use \"pgformatter\"" pgformatter)
-                 (const :tag "Use \"sqlfluff\"" sqlfluff)))
+                 (const :tag "Use \"sqlfluff\"" sqlfluff)
+                 (const :tag "Use \"sql-formatter\"" sql-formatter))))
```
Comment (purcell) [inline sqlformat.el]: CI says there's an issue with matching parens somewhere around here. :)

**Review by purcell — CHANGES_REQUESTED** [review]
Thanks, looks good, but CI is failing due to a syntax error.

File: sqlformat.el (hunk 1 above)
Comment (Trisk3lion) [reply sqlformat.el]: Sorry for that, fixed now!

That's what you get for trying to make changes in the github editor instead of emacs...

**Comment by purcell** [comment]
Great, thanks!

File: sqlformat.el (hunk 1 above)
Comment (purcell) [reply sqlformat.el]: > That's what you get for trying to make changes in the github editor instead of emacs...

Yup, been there, done that... 😂

**Pull request merged**